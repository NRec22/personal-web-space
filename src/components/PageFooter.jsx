import React from 'react';
import Footer from 'react-bulma-components/lib/components/footer';
import Container from 'react-bulma-components/lib/components/container';
import Content from 'react-bulma-components/lib/components/content';

const PageFooter = () => (
  <Footer>
    <Container>
      <Content>
        <p>Powered by React and Bulma.</p>
      </Content>
    </Container>
  </Footer>
);

export default PageFooter;