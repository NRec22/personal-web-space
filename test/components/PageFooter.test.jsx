import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import Footer from 'react-bulma-components/lib/components/footer';
import PageFooter from '../../src/components/PageFooter';

describe('Footer: ', () => {
  it('contains a bulma footer', () => {
    const wrapper = shallow(<PageFooter />);
    expect(wrapper.find(Footer)).toHaveLength(1);
  })

  it('should render as expected', () => {
    const wrapper = shallow(<PageFooter />);
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  })
})