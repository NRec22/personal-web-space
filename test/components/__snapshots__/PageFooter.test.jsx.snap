// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer:  should render as expected 1`] = `
<Footer
  className=""
  renderAs="div"
  style={Object {}}
>
  <Container
    breakpoint={null}
    className=""
    fluid={false}
    renderAs="div"
    style={Object {}}
  >
    <Content
      className=""
      renderAs="div"
      size={null}
      style={Object {}}
    >
      <p>
        Powered by React and Bulma.
      </p>
    </Content>
  </Container>
</Footer>
`;
